### PR TITLE
crow coupler, labels on crow rings

### PR DESCRIPTION
**Pull request by simbilod**
- Crow variant with couplers
- All bends in crow components are labeled for easier access later (for instance to connect to actuators)

image

### CODE REVIEW

**Comment by simbilod** [comment]
Somehow I cannot currently run the test to regenerate the component, I am getting ERROR  - ImportError: numpy.core.multiarray failed to import

File: gdsfactory/components/ring_crow.py [hunk 1]
```diff
@@ -73,16 +73,18 @@ def ring_crow(
...
         gap = gf.snap.snap_to_grid(gap, nm=2)
         ring = Component()
         bend_c = gf.get_component(bend, radius=r, cross_section=cross_section)
         xs = gf.get_cross_section(cross_section)
         bend_width = xs.width
-        bend1 = ring.add_ref(bend_c)
-        bend2 = ring.add_ref(bend_c)
-        bend3 = ring.add_ref(bend_c)
-        bend4 = ring.add_ref(bend_c)
+        bend1 = c.add_ref(bend_c, alias=f"bot_right_bend_ring_{index}")
```
[the remark this answers is not in the document]
Comment (simbilod) [reply gdsfactory/components/ring_crow.py:84]: Strange, thanks for the heads up I will take a look

[the remark this answers is not in the document]
Comment (joamatab) [reply gdsfactory/components/ring_crow.py:84]: I fixed in a PR

[the remark this answers is not in the document]
Comment (simbilod) [reply gdsfactory/components/ring_crow.py:84]: Thanks!

**Pull request merged**